KoAR.SaveEditor/App.xaml.cs

```diff
@@ -19,16 +19,20 @@ partial class App
         public static void ShowExceptionDialog(string mainInstruction, Exception exception)
         {
             string content = $"{exception.GetType().FullName}: {exception.Message}";
-            TaskDialogResult dialogResult = TaskDialog.Show(new()
+            TaskDialog dialog = new(new()
             {
                 Title = "KoAR Save Editor",
-                MainInstruction = mainInstruction,
-                Content = content,
-                ExpandedInfo = exception.StackTrace,
-                VerificationText = "Open GitHub bug report? (requires free account)",
-                MainIcon = VistaTaskDialogIcon.Error,
+                Instruction = mainInstruction,
+                Text = content,
+                Expander = {
+                    Text = exception.StackTrace,
+                    ExpandFooterArea = true,
+                },
+                CheckBox = new("Open GitHub bug report? (requires free account)"),
+                Icon = TaskDialogStandardIcon.Error,
             });
-            if (dialogResult.VerificationChecked == true)
+            dialog.Show();
+            if (dialog.Page.CheckBox.Checked)
             {
                 string title = $"{content} (in v{App.Version})";
                 Process.Start($"https://github.com/mburbea/koar-item-editor/issues/new?labels=bug&template=bug_report.md&title={WebUtility.UrlEncode(title)}");
```

Comment:
You need to set UseShellExecute = true in .net core for this to work, hence use 

```
  ProcessStartInfo info = new($"http://blah") { UseShellExecute = true };
  Process.Start(info)?.Dispose();
```

Reply:
See: https://github.com/dotnet/runtime/issues/28005

Reply:
That's annoying, but I can make that change. This is still net48.